src/main/java/com/epam/ms/util/Deserializer.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.ms.util;
```

Comment:
I have just noticed `util` folder..
to be honest, I don't really like Util and they used to tell me to try to avoid such folder in the projects. It can contain everything.. 
I'd propose to have a kind of `service` folder and create more `XXXService` classes. For instance, there you can define `EmailService`
`Deserializer` is used only in `RabbitMqListener` class. You can either put `Deserializer` class in the same folder or even create this `deserialize` method inside `RabbitMqListener` and don't maintain an additional `Deserializer` class

Reply:
Thank you, done